### DISCUSSION TITLE
[Mobile]: Bug when logging in inside vscode.dev.

### DISCUSSION THREAD

**Discussion by RyanNorooz** [community/community#61208] [Mobile]
I can't login into my GitHub account!
I'm using an Android 12 device.
Inside vscode.dev, When trying to turn on settings sync and using GitHub to login in, the GitHub mobile app opens up and shows a list of browsers to continue the action.
Regardless of the choice, nothing happens in vscode.dev.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @RyanNorooz ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by moonfury-ops** [comment]
Thank you

**Comment by dannz510** [comment]
Here are some general suggestions that might help:

1. Make sure that you have the latest version of the GitHub mobile app installed on your device.
2. Try clearing the cache and cookies of your browser and the GitHub app.
3. Check if there are any updates available for your browser or device.
4. Try using a different browser to see if the issue persists.
If the issue still persists, you may want to reach out to the [support teams of vscode.dev](https://visualstudio.microsoft.com/fr/support/) or GitHub for further assistance.
